ipyleaflet/leaflet.py

```diff
@@ -747,6 +749,13 @@ def clear_markers(self):
         self.send({'msg': 'clear_markers'})
 
 
+class MapStyle(Style, Widget):
+    """ Map Style Widget """
+    _model_name = Unicode('LeafletMapStyleModel').tag(sync=True)
+    _model_module = Unicode("jupyter-leaflet").tag(sync=True)
+    cursor = Unicode(help='Cursor type').tag(sync=True)
```

Comment:
Maybe add the default `'move'` value here? At some point, we could use an `Enum` for allowing only a range of values, but you can leave it as it is for now :)

Reply:
I tried that, but as nothing changed I left it as is..